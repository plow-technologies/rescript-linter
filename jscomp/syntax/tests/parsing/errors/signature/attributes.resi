module type MissingItem = {
  @attr
}

@attrWithoutItem
