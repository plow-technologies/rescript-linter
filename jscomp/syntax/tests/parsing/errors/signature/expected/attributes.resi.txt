
  Syntax error!
  tests/parsing/errors/signature/attributes.resi:2:3-7

  1 │ module type MissingItem = {
  2 │   @attr
  3 │ }
  4 │ 

  Did you forget to attach `attr` to an item?
  Standalone attributes start with `@@` like: `@@attr`


  Syntax error!
  tests/parsing/errors/signature/attributes.resi:5:1-16

  3 │ }
  4 │ 
  5 │ @attrWithoutItem
  6 │ 

  Did you forget to attach `attrWithoutItem` to an item?
  Standalone attributes start with `@@` like: `@@attrWithoutItem`

module type MissingItem  = sig [%%rescript.sigitemhole ] end
[%%rescript.sigitemhole ]